src/amo/components/SearchContextCard/index.js

```diff
@@ -83,6 +83,15 @@ export class SearchContextCardBase extends React.Component<InternalProps> {
               ),
               { count: i18n.formatNumber(count), query },
             );
+          } else if (tag) {
```

Comment:
Will `addonType` be set somewhere?

Reply:
it can be set with the filters on the page